setup.py

```diff
@@ -154,14 +154,15 @@ def run(self):
         'install':   install_command,
     },
 
-    author='The lovely Arbor devs.',
+    author='The Arbor dev team.',
     url='https://github.com/arbor-sim/arbor',
     description='High performance simulation of networks of multicompartment neurons.',
     long_description=long_description,
     long_description_content_type='text/markdown',
     classifiers=[
-        'Development Status :: 4 - Beta', # Upgrade to "5 - Production/Stable" on release of v0.4
+        'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Science/Research',
+        "License :: OSI Approved :: BSD License",
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
```

Comment:
I do not know whehter we should add C++/CUDA here, eg numpy adds C to the list.

Reply:
We could. I find these things really fiddly to get working properly.

Reply:
I added C++ and CUDA 10 & 11

Reply:
👍